README.md

```diff
@@ -1,2 +1,102 @@
 # node-kafka-client
-A node-rdkafka wrapper  with extended features
+
+A wrapper library of [node-rdkafka](https://github.com/Blizzard/node-rdkafka) with extended features
+
+## Install
+
+> Important: We are using `node-rdkafka^2.7.1-2` which is not supporting Node v12 or later. Let's use Node version v8 > v10
+
+`npm install @uiza/node-kafka-client`
+
+## Usage
+
+### Consumer
+
+```js
+const { Consumer } = require("@uiza/node-kafka-client");
+
+const consumer = new Consumer({
+  // Required options
+  name: "test-consumer-name",
+  groupId: "test-group-id",
+  host: "localhost:9092",
+  topic: "test-topic",
+
+  // Optional options
+  connectTimeout: 5000,
+  mode: "non-flowing",
+  intervalFetchMessage: 10, // Only affect for non-flowing mode
```

Comment:
Should describe the unit of this parameter is using second/millisecond for more clearly.